packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -1498,12 +1503,22 @@ function rerenderDeferredValue<T>(
 
 function startTransition(setPending, config, callback) {
   const priorityLevel = getCurrentPriorityLevel();
+  const previousLanePriority = getCurrentUpdateLanePriority();
+  setCurrentUpdateLanePriority(InputContinuousLanePriority);
```

Comment:
Sorry, I misspoke. If the previous priority is higher than InputContinuousLanePriority, then we shouldn't lower it.